modules/op-approving-task-web-console.adoc

```diff
@@ -0,0 +1,28 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-approving-task-web-console_{context}"]
+= Approving a manual approval task by using the web console
+
+You can approve or reject a manual approval task by using the {OCP} web console.
+
+If you are listed as an approver in a manual approval task and a pipeline run reaches this task, the web console a notification. You can view a list of tasks that require your approval and approve or reject these tasks.
+
+.Prerequisites
+
+* You enabled the {pipelines-shortname} console plugin.
```

Comment:
Can we specify the link here how to enable dynamic plugin ?

Reply:
The link to the section about installing the plugin is specified in "Additional resources" at the end of this section, but in the PR it is in the assembly file - this is a limitation of the OpenShift documentation tooling.

Reply:
Got it. Then its fine.